src/scirpy/tl/_mutational_load.py

```diff
@@ -0,0 +1,227 @@
+                cdr2_germline = airr_df.iloc[row].loc[f"{chain}_{germline_alignment}"][165:195]
+                fwr3_germline = airr_df.iloc[row].loc[f"{chain}_{germline_alignment}"][195:312]
+                cdr3_germline = airr_df.iloc[row].loc[f"{chain}_{germline_alignment}"][
+                    312 : (312 + airr_df.iloc[row].loc[f"{chain}_junction_len"] - 6)
+                ]
+                fwr4_germline = airr_df.iloc[row].loc[f"{chain}_{germline_alignment}"][
+                    (312 + airr_df.iloc[row].loc[f"{chain}_junction_len"] - 6) :
+                ]
+
+                mutation_dict["fwr1"].append(
+                    simple_hamming_distance(
+                        subregion_df.iloc[row].loc[f"{chain}_fwr1"],
+                        fwr1_germline,
+                        frequency=frequency,
+                        ignore_chars=ignore_chars,
+                    )
+                )
+                mutation_dict["cdr1"].append(
+                    simple_hamming_distance(
+                        subregion_df.iloc[row].loc[f"{chain}_cdr1"],
+                        cdr1_germline,
+                        frequency=frequency,
+                        ignore_chars=ignore_chars,
+                    )
+                )
+                mutation_dict["fwr2"].append(
+                    simple_hamming_distance(
+                        subregion_df.iloc[row].loc[f"{chain}_fwr2"],
+                        fwr2_germline,
+                        frequency=frequency,
+                        ignore_chars=ignore_chars,
+                    )
+                )
+                mutation_dict["cdr2"].append(
+                    simple_hamming_distance(
+                        subregion_df.iloc[row].loc[f"{chain}_cdr2"],
+                        cdr2_germline,
+                        frequency=frequency,
+                        ignore_chars=ignore_chars,
+                    )
+                )
+                mutation_dict["fwr3"].append(
+                    simple_hamming_distance(
+                        subregion_df.iloc[row].loc[f"{chain}_fwr3"],
+                        fwr3_germline,
+                        frequency=frequency,
+                        ignore_chars=ignore_chars,
+                    )
+                )
+                mutation_dict["cdr3"].append(
+                    simple_hamming_distance(
+                        subregion_df.iloc[row].loc[f"{chain}_cdr3"],
+                        cdr3_germline,
+                        frequency=frequency,
+                        ignore_chars=ignore_chars,
+                    )
+                )
+                mutation_dict["fwr4"].append(
+                    simple_hamming_distance(
+                        subregion_df.iloc[row].loc[f"{chain}_fwr4"],
+                        fwr4_germline,
+                        frequency=frequency,
+                        ignore_chars=ignore_chars,
+                    )
+                )
```

Comment:
I think this could be further simplified by 

(1)
defining a dict of regions
```python
regions = {
  "fwr1": (0, 78),
  "cdr1": (78, 114),
  ...
}
```

and then looping through it; somewhat like

```python
mutation_dict = {}
for region, coordinates in regions.items():
    mutation_dict[region] = simple_hamming_distance(
       subregion_df.iloc[row].loc[f"{chain}_fwr3"],
       airr_df.iloc[row].loc[f"{chain}_{germline_alignment}"][slice(*coordinates)]
       ...
   )
```

Reply:
I see :D
Thank's for the suggestions! I will try to further simplify it!

Reply:
Thanks for the hint. I just adapted the code based on this suggestion and now it looks so much better :D

What's still pending is a robust test-case. I had a look on how other tests are written and I get the idea. However, I still don't know what's the best way to get/generate test-data. Should I manually generate a small (e.g. 10 sequences) dataset inside the test-function, which is used to test mutational_load or what is the best-practice here?
The dataset needs to be IMGT numbered so I don't think that I could load any of scirpy's native datasets here...

Reply:
>  Should I manually generate a small (e.g. 10 sequences) dataset inside the test-function

Yes, this is common practice. You can also put a small data file in `src/scirpy/tests/data`.

Reply:
Oke, I'll come up with something for my next push here :)